projects/plugins/boost/app/assets/src/js/pages/settings/sections/Modules.svelte

```diff
@@ -48,7 +48,19 @@
 
 	const suggestRegenerate = suggestRegenerateDS.store;
 
-	async function resume() {
+	$: if ( $modulesState.critical_css.available && $modulesState.critical_css.active ) {
```

Comment:
I thought for this too long and now I'm questioning whether the available check should be here in the first place 🤔  - it should never be active if it's not available 🤔

Reply:
I had the same thought too, but I didn't check if that's the case for some reason.

Now that you mentioned it, I did check and you're right - a module can't be unavailable and active at the same time. It's can only be active if it's available.

Done in a50757f.